<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountPage</class>
 <widget class="QWidget" name="AccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>981</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Cart">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>Cart</string>
    </attribute>
    <widget class="QListWidget" name="ProductList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>651</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Nirmala UI&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ItemImage">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>userIcon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="RemoveFromList">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove From List</string>
     </property>
    </widget>
    <widget class="QLabel" name="ItemPrice">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>370</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$50.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Products</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="ItemRating">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>430</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Rating: 4/5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="History">
    <attribute name="title">
     <string>History</string>
    </attribute>
    <widget class="QLabel" name="Title_h">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Products</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QListWidget" name="ProductList_h">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>651</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Nirmala UI&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ItemPrice_h">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>370</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$50.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="ItemImage_h">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>userIcon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="BuyAgain">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy Again</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReviewProduct">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>440</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Review Product</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Track">
    <attribute name="title">
     <string>Track Order</string>
    </attribute>
    <widget class="QListWidget" name="ProductList_t">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>651</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Nirmala UI&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item 3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="Title_t">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Products</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="TextBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string>No Items Available</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelOrder">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel Order</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="CustomerName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Md Sakil Khan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceData">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance: $100.00 Chase Bank *1234</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>userIcon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerInfo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123-12 123st, Qwerty, NY 12345</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Logout">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
